weather/www/js/app.js

```diff
@@ -0,0 +1,147 @@
+
+    $scope.hour1 = time(1) + ", " + Math.round($scope.weather.hourly.data[1].temperature) + "°F and " +
+      $scope.weather.hourly.data[1].summary;
+    $scope.hour2 = time(2) + ", " + Math.round($scope.weather.hourly.data[2].temperature) + "°F and " +
+      $scope.weather.hourly.data[2].summary;
+    $scope.hour3 = time(3) + ", " + Math.round($scope.weather.hourly.data[3].temperature) + "°F and " +
+      $scope.weather.hourly.data[3].summary;
+    $scope.hour4 = time(4) + ", " + Math.round($scope.weather.hourly.data[4].temperature) + "°F and " +
+      $scope.weather.hourly.data[4].summary;
+    $scope.hour5 = time(5) + ", " + Math.round($scope.weather.hourly.data[5].temperature) + "°F and " +
+      $scope.weather.hourly.data[5].summary;
+    $scope.hour6 = time(6) + ", " + Math.round($scope.weather.hourly.data[6].temperature) + "°F and " +
+      $scope.weather.hourly.data[6].summary;
+    $scope.hour7 = time(7) + ", " + Math.round($scope.weather.hourly.data[1].temperature) + "°F and " +
+      $scope.weather.hourly.data[7].summary;
+    $scope.hour8 = time(8) + ", " + Math.round($scope.weather.hourly.data[2].temperature) + "°F and " +
+      $scope.weather.hourly.data[8].summary;
+    $scope.hour9 = time(9) + ", " + Math.round($scope.weather.hourly.data[3].temperature) + "°F and " +
+      $scope.weather.hourly.data[9].summary;
+    $scope.hour10 = time(10) + ", " + Math.round($scope.weather.hourly.data[4].temperature) + "°F and " +
+      $scope.weather.hourly.data[10].summary;
+    $scope.hour11 = time(11) + ", " + Math.round($scope.weather.hourly.data[5].temperature) + "°F and " +
+      $scope.weather.hourly.data[11].summary;
+    $scope.hour12 = time(12) + ", " + Math.round($scope.weather.hourly.data[6].temperature) + "°F and " +
+      $scope.weather.hourly.data[12].summary;
+
+    $scope.day1 = dayOfTheWeek(1) + ", " + Math.round($scope.weather.daily.data[1].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[1].summary;
+    $scope.day2 = dayOfTheWeek(2) + ", " + Math.round($scope.weather.daily.data[2].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[2].summary;
+    $scope.day3 = dayOfTheWeek(3) + ", " + Math.round($scope.weather.daily.data[3].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[3].summary;
+    $scope.day4 = dayOfTheWeek(4) + ", " + Math.round($scope.weather.daily.data[4].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[4].summary;
+    $scope.day5 = dayOfTheWeek(5) + ", " + Math.round($scope.weather.daily.data[5].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[5].summary;
+    $scope.day6 = dayOfTheWeek(6) + ", " + Math.round($scope.weather.daily.data[6].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[6].summary;
+    $scope.day7 = dayOfTheWeek(7) + ", " + Math.round($scope.weather.daily.data[7].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[7].summary;
+
```

Comment:
I think you should be able to do all of this stuff in a loop. You could do something along the lines of putting the variables in an array, and then assigning them values via looping thru the array. Something like:
var array = [$scope.day1, $scope.day2, ... ]
for (i = 1; i < 8; i++) {
    array[i] = dayOfTheWeek(i) + ", " + Math.round($scope.weather.daily.data[i].temperatureMax) + "°F and " +
 +      $scope.weather.daily.data[i].summary;
}

Or it might have to be something like this: 

var array = [day1, day2, ... ]
for (i = 1; i < 8; i++) {
    $scope.array[i] = dayOfTheWeek(i) + ", " + Math.round($scope.weather.daily.data[i].temperatureMax) + "°F and " +
 +      $scope.weather.daily.data[i].summary;
}

I'm definitely not sure exactly how to do this, but something along those lines would work, I think.